### ISSUE TITLE
Filtering Approach

### ISSUE THREAD

**Issue by david-dayan** [State-Fisheries-Genomics-Lab/GT-seq#8]
After the March 8th meeting we agreed on changing the filtering approach to first remove very problematic/missing individuals, genotypes, then arrive at the the final filtering cutoffs. this iterative filtering approach is also covered in issue #2

**Comment by david-dayan** [comment]
another note: we agreed to write some code to flag potential paralogs, in addition to using the moderate missingness flag as we did before

**Comment by david-dayan** [comment]
Totally updated the filtering approach. 

Changes:

1. Iterative filtering: each filtering step is completed in sequence (not at the same time), and there are two rounds of filtering (a first round that removes only the very worst individuals and loci, then a second round that brings us to our final filtering levels)
2. Increased types of "bad markers" to check for paralogs. Previously we looked at the allele count ratio for markers with moderate but not severe missingness (10-20%). Added some lines of code that will identify markers with highly skewed and highly variable allele count ratios at heterozygotes and "in-betweeners"
